wp-content/themes/cf-voyager/lessons-cpt.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * Template Name: Lessons CPT
+ *
+ */
+
+get_header();
+
+$custom_terms = get_terms('lesson_types');
```

Comment:
For the specific requirements of the #5 issue, you do not need to get the list of terms.